src/EditorFeatures/Test2/Rename/RenameTagProducerTests.vb

```diff
@@ -840,7 +840,7 @@ class Goo
             End Using
         End Function
 
-        <WpfFact>
+        <WpfFact(Skip:="https://github.com/dotnet/roslyn/issues/38247")>
```

Comment:
Please do not skip this test. It's flakiness level does not fall below the bar to disable it, it complicates the process to fix, and the particular issue impacts several different tests even though only one is skipped here.

Reply:
This test is currently meeting our bar for flakiness. It's responsible for a significant number of CI build failures. Disabling it is approrpriate. If you want to re-unable it, with a corresponding fix, please feel free to do so.

Reply:
This test has a 30 day success rate of 99.94%. It is the least flaky test of all tests in the entire report.